Added Brunt config flow (#20345)
home-assistant/home-assistant.github.io: source/_integrations/brunt.markdown

ha_domain: brunt
ha_platforms:
  - cover
ha_config_flow: true
---

The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.

## Configuration

{% include integrations/config_flow.md %}

> {% configuration_basic %}

Reviewer: I think we normally remove the YAML config docs when adding config flow and deprecating YAML.

Author: yeah, wasn't sure, deleted now.

ha_domain: brunt
ha_platforms:
  - cover
ha_config_flow: true
---

The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.

> ## Configuration

Reviewer: Check the preview build linked in the CI checks. This header is a duplicate. It's already included by the include below.

ha_domain: brunt
ha_platforms:
  - cover
ha_config_flow: true
---

> The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.

Reviewer: ```suggestion
The `brunt` integration allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use it, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.
```
